benchmarks/bench_mixed_attention.py

```
@@ -72,6 +72,24 @@ def run_bench(
+        torch.arange(num_blocks, dtype=torch.int32, device=device),
+        seq_lens.to(device),
+        num_qo_heads,
+        num_kv_heads,
+        head_dim,
+        head_dim,
+        page_block_size,
+        causal=causal,
+        q_data_type=torch.bfloat16,
+        kv_data_type=torch.bfloat16,
+    )
+    o_persistent, _ = wrapper_persistent.run(q, kv_data)
```

Ruff (0.14.2)

Unpacked variable `o_persistent` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)